tests/cloudpickle_test.py

```diff
@@ -521,7 +521,7 @@ def test_module_locals_behavior(self):
         pickled_func_path = os.path.join(self.tmpdir, 'local_func_g.pkl')
 
         child_process_script = '''
-        import pickle
+        from cloudpickle.compat import pickle
```

Comment:
using `cloudpickle.compat.pickle` or `pickle` should be equivalent in the `load` case right?